pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -223,26 +233,24 @@ task train_classifier {
         File    taxanomy_ref
         String  forward_adapter
         String  reverse_adapter
-        Int?    min_length  =   100
-        Int?    max_length  =   500
+        Int    min_length  =   100
+        Int    max_length  =   500
```

Comment:
These were made to be mandatory but doesn't seem like they're used that way below?

Is 100/500 our/Zoe's custom default, or is it the tool's default if we don't specify the parameter?